pyproject.toml

```diff
@@ -8,14 +8,14 @@ license = "MIT"
 [tool.poetry.dependencies]
 APScheduler = "3.10.4"
 cachetools = "^5.3.1"
-fastapi = "^0.88.0"
+fastapi = "^0.115.0"
 framegrab = "^0.5.0"
-groundlight = "^0.17.6"
+groundlight = "<0.18.0"
```

Comment:
Needed until the endpoint is updated to handle non-binary modes

Reply:
See my comment on your other PR - we should really be pinning to a specific version here